plugins/processors/starlark/testdata/rename_measurement.star

```diff
@@ -0,0 +1,14 @@
+# Rename the measurement name to the fieldname. Rename the fieldname to value. 
+#
+# Example Input:
+# prometheus_remote_write,instance=localhost:9090,job=prometheus,quantile=0.99 go_gc_duration_seconds=4.63 1614889298859000000
+#
+# Example Output:
+# go_gc_duration_seconds,instance=localhost:9090,job=prometheus,quantile=0.99 value=4.63 1614889299000000000
+
+def apply(metric):
+   for k, v in metric.fields.items():
+       metric.name = k
+       metric.fields["value"] = v
+       metric.fields.pop(k)
+   return metric
```

Comment:
if we're going to recommend this to everyone, we should include a metric.name guard in this function to look for the metric name, and either handle the case where there are multiple fields (they can't all be the metric name), or specify in the description and file name that this is specifically for prometheus.

Reply:
Added a guard to check it's promtheus_remote_write. I'd put in the other example a namepass but probably makes more sense to have it in there. I've added in to the name and description that it's for promtheus remote write where there is only one field as that is the use case.